sentry-core/src/main/java/io/sentry/core/protocol/DebugImage.java

```diff
@@ -94,6 +95,6 @@ public void setCodeId(String codeId) {
   @ApiStatus.Internal
   @Override
   public void acceptUnknownProperties(Map<String, Object> unknown) {
-    this.unknown = unknown;
+    this.unknown = new ConcurrentHashMap<>(unknown);
```

Comment:
If this type isn't used by `Scope`, it doesn't need to be thread-safe.
It is easier to reason about it to just make them so.
Manoel, what do you think?

Reply:
it could be a non-thread-safe hashmap

Reply:
Agreed, I'll remove it from classes that aren't used in Scope

Reply:
Fixed 👍